drivers/azure/azure.go

```diff
@@ -196,6 +201,12 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			EnvVar: "AZURE_AVAILABILITY_SET",
 			Value:  defaultAzureAvailabilitySet,
 		},
+		mcnflag.StringFlag{
+			Name:   flAzureNSG,
+			Usage:  "Azure Network Security Group to assign this node to (default is to create a new NSG for just this machine)",
```

Comment:
```suggestion
			Usage:  "Azure Network Security Group to assign this node to (default is to create a new NSG for each machine)",
```

Reply:
updated in latest commit

Reply:
After our offline discussion, can we update to say something like `accepts either a name or resource ID`

Reply:
updated in latest commit